src/components/footer/Companies.tsx

```diff
@@ -10,12 +11,14 @@ interface FooterCompaniesProps {
 
 const FooterCompanies: FC<FooterCompaniesProps> = ({ config }) => {
   return (
-    <Flex
+    <Box
+      gap={{ base: 'md', md: '2xl' }}
```

Comment:
It only makes the trick in new browsers. In the old ones we will see:

**Desktop**

![Screenshot 2023-03-10 at 17 10 06](https://user-images.githubusercontent.com/37380787/224373357-ed187080-223d-4479-8e7c-182de5b3a095.png)

**Moile**
![Screenshot 2023-03-10 at 17 44 31](https://user-images.githubusercontent.com/37380787/224373543-bf2ce409-964c-4955-b250-de6bced329ce.png)